chains/container-chains/runtime-templates/simple/src/lib.rs

```diff
@@ -32,7 +34,7 @@ pub use sp_runtime::BuildStorage;
 pub mod migrations;
 pub mod weights;
 
-pub use sp_runtime::{traits::Extrinsic as ExtrinsicT, MultiAddress, Perbill, Permill};
+pub use sp_runtime::{traits::ExtrinsicLike, MultiAddress, Perbill, Permill};
```

Comment:
So, they renamed Extrinsic trait to ExtrinsicLike. Now we don't have to do import aliasing everywhere.